packages/trader/src/Modules/Reports/Constants/data-table-constants.js

```diff
@@ -117,12 +117,7 @@ export const getProfitTableColumnsTemplate = (currency, items_count) => [
         title: localize('Sell time'),
         col_index: 'sell_time',
         renderHeader: ({ title }) => {
-            return (
-                <>
-                    <span>{title}</span>
-                    <Icon icon='IcContractFlag' />
-                </>
-            );
+            return <span>{title}</span>;
         },
```

Comment:
Can we keep it in one line?
```js
renderHeader: ({ title }) => <span>{title}</span>;
```

Reply:
@mahdiaryayi-fs cool, I am amazed why I didn't notice it. just fixed it